en/dev_log/log_encryption.md

```diff
@@ -0,0 +1,165 @@
+# Log Encryption
+
+The [system logger](../modules/modules_system.md#logger) is able to provide encrypted logs which can be decrypted manually before analysis. Currently, to have the logger create encrypted logs, you will need to flash firmware on your flight controller with this functionality supported. 
+
+::: info
+Currently, the only build that has log encryption enabled by default is `px4-fmu-v5_cryptotest`. If you wish to add it to other flight controllers, it will require you to modify the build to enable this feature. See below for instructions on how to do that.
+:::
+
+The default encryption algorithm used is XChaCha20, and the default wrapping algorithm used is RSA2048-OAEP. AES is also supported. The encryption algorithm used can be changed by modifying [SDLOG_ALGORITHM](../advanced_config/parameter_reference.md#SDLOG_ALGORITHM).
```

Comment:
I wouldn't mention AES here; It doesn't work with current upstream PX4. It is a placeholder where one can add an own AES backend implementation.

Reply:
FYI both, I have just removed that sentence.

Reply:
Thank you!

Reply:
Can we have a modification to the source code for `SDLOG_ALGORITHM` to indicate that it is currently not supported?

Reply:
FYI, I have added https://github.com/PX4/PX4-Autopilot/pull/23816 to make it clear the AES option is not supported yet.